win32ss/drivers/videoprt/videoprt.c

```diff
@@ -1299,8 +1299,62 @@ VideoPortCreateSecondaryDisplay(
     IN OUT PVOID *SecondaryDeviceExtension,
     IN ULONG Flag)
 {
-    UNIMPLEMENTED;
-    return ERROR_DEV_NOT_EXIST;
+    WCHAR DeviceVideoBuffer[20];
+    UNICODE_STRING DeviceName;
+    WCHAR SymlinkBuffer[20];
+    UNICODE_STRING SymlinkName;
+    PDEVICE_OBJECT DeviceObject;
+    PVIDEO_PORT_DEVICE_EXTENSION FirstDeviceExtension, DeviceExtension;
+    NTSTATUS Status;
+
+    ASSERT(SecondaryDeviceExtension);
+
+    if (Flag != 0)
+    {
+        UNIMPLEMENTED;
+    }
+
+    FirstDeviceExtension = VIDEO_PORT_GET_DEVICE_EXTENSION(HwDeviceExtension);
+
+    Status = IntVideoPortCreateAdapterDeviceObject(FirstDeviceExtension->DriverObject,
+                                                   FirstDeviceExtension->DriverExtension,
+                                                   NULL,
+                                                   &DeviceObject);
+    if (!NT_SUCCESS(Status))
+    {
+        DPRINT1("IntVideoPortCreateAdapterDeviceObject() failed with status 0x%08x\n", Status);
+        return ERROR_DEV_NOT_EXIST;
+    }
+
+    DeviceExtension = DeviceObject->DeviceExtension;
+    VideoPortDeviceNumber++;
+
+    /* Add entry to DEVICEMAP\VIDEO key in registry. */
+    swprintf(DeviceVideoBuffer, L"\\Device\\Video%d", DeviceExtension->DeviceNumber);
+    RtlWriteRegistryValue(
+        RTL_REGISTRY_DEVICEMAP,
```

Comment:
https://reactos.org/wiki/Coding_Style#Indentation_and_line_width
point 4.

Reply:
I remember that this part of the Coding Style wiki page was brought in by @Extravert-ir in the November 2020 meeting and we voted on it.

I'm not sure though if the intention was to align function arguments at all if they don't fit into a single line, or if we're pedantic here and only accept a single style for aligning function arguments in new code.
I would be fine either way, but I'd like to hear what @Extravert-ir's intention was.

Reply:
Done